src/patterns/gender-and-sex/index.md.njk

```diff
@@ -0,0 +1,50 @@
+---
+title: Gender and sex
+description: Understand when and how to ask users for their gender or sex. 
+section: Patterns
+theme: Ask users for...
+aliases:
+backlog_issue_id: 69
+layout: layout-pane.njk
+---
+
+{% from "_example.njk" import example %}
+
+Understand when and how to ask users for their gender or sex. 
+
+## When to use this pattern
+
+Avoid asking users for their gender or sex. 
+
+Only ask about gender or sex if you genuinely can’t deliver your service without this information. 
+
+If you do need to ask, use ‘sex’ when you need biological data (for example, if you’re providing a medical service). In all other cases, use ‘gender’. 
+
+## How it works
+
+If you have to ask about gender, you should:
+
+- always say why you’re asking and what you’ll do with the information
+- list the fields in alphabetical order, for example, ‘Female’, ‘Male’, ‘Nonbinary’
+- do research to test that this works for your users
+
+{{ example({group: 'patterns', item: 'gender-and-sex', example: 'default', html: true, nunjucks: true, open: false}) }}
+
+### Avoid using gendered pronouns
+
+You should address the user as ‘you’ where possible and avoid using gendered pronouns like ‘he’ and ‘she’.
+
+You may not always be able to use ‘you’. For example, if your service allows people to jointly apply for something, you might need to refer to both the user and the person they’re applying with. In these cases, use the person’s name, ‘they’ or ‘their’.
+
+### Don’t try to guess gender or sex
+
+Never try to guess someone’s gender or sex based on personal characteristics such as their name, title, appearance or voice.
+
+These characteristics are not reliable indicators of gender or sex. If you really need this information to deliver your service, only collect it by asking users directly.
+
+You should also [avoid asking for someone’s title](https://www.gov.uk/service-manual/design/names#avoid-asking-for-peoples-title).
```

Comment:
Should this link to https://govuk-design-system-production.cloudapps.digital/patterns/names/#avoid-asking-for-a-person-s-title?

Reply:
Fixed